apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -34,165 +30,150 @@ import * as questionServers from '../../question-servers/index.js';
-  });
+  if (rubric_items.length > 0) {
+    let rubric_info = '';
+    for (const item of rubric_items) {
+      rubric_info += `description: ${item.description}\n`;
+      if (item.explanation) {
+        rubric_info += `explanation: ${item.explanation}\n`;
+      }
+      if (item.grader_note) {
+        rubric_info += `grader note: ${item.grader_note}\n`;
+      }
+      rubric_info += '\n';
+    }
+    messages.push({
+      role: 'system',
+      content:
+        "You are an instructor for a course, and you are grading assignments. You are provided several rubric items with a description, explanation, and grader note. You must grade the assignment by using the rubric and returning an object of rubric descriptions and whether or not that rubric item applies to the student's submission. If no rubric items apply, do not select any. I will provide some example answers and their corresponding selected rubric items.",
+    });
+    messages.push({
+      role: 'system',
+      content: `Here are the rubric items:\n\n${rubric_info}`,
+    });
+  } else {
+    messages.push({
+      role: 'system',
+      content:
+        'You are an instructor for a course, and you are grading assignments. You should always return the grade using a JSON object with two properties: score and feedback. The score should be an integer between 0 and 100, with 0 being the lowest and 100 being the highest. The feedback should explain why you give this score. Follow any special instructions given by the instructor in the question. Omit the feedback if the answer is correct. I will provide some example answers and their corresponding scores and feedback.',
+    });
+  }
 
   // Question prompt
   messages.push({
     role: 'user',
-    content: `Question: \n${question_prompt}`,
+    content: `Question: \n${questionPrompt}`,
   });
 
   // Examples
   for (const example of example_submissions) {
-    messages.push({
-      role: 'user',
-      content: `Example answer: \n${example.submission_text} \nGrade to this example answer: \n${example.score_perc}`,
-    });
+    if (rubric_items.length > 0 && example.manual_rubric_grading_id) {
+      const rubric_grading_items = await queryRows(
+        sql.select_rubric_grading_items,
+        {
+          manual_rubric_grading_id: example.manual_rubric_grading_id,
+        },
+        RubricItemSchema,
+      );
+      // Warning when graded rubric item does not match current rubric item
+      if (
+        rubric_grading_items.length > 0 &&
+        rubric_grading_items[0].rubric_id !== rubric_items[0].rubric_id
+      ) {
+        warning += `Instance question ${example.instance_question_id}: example rubric id is different from the current rubric id.\n`;
```

Comment:
I didn't trace through where these warnings are going, but do we really want this one? I tend to think not. If the rubric has had all items deleted and new ones added, it's just like this situation but we won't warn, so I think the warning doesn't make much sense.